wdl-format/src/config/max_line_length.rs

```diff
@@ -0,0 +1,52 @@
+//! Configuration for max line length formatting.
+
+/// The error type for max line length configuration.
+pub enum Error {
+    /// The max line length is invalid.
+    Invalid(&'static str),
+}
+
+impl std::fmt::Display for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Error::Invalid(field) => write!(
+                f,
+                "invalid value for '{field}' in a formatter configuration builder"
+            ),
+        }
+    }
+}
+
+/// The default maximum line length.
+pub const DEFAULT_MAX_LINE_LENGTH: usize = 90;
+/// The minimum maximum line length.
+pub const MIN_MAX_LINE_LENGTH: usize = 60;
+/// The maximum maximum line length.
+pub const MAX_MAX_LINE_LENGTH: usize = 240;
```

Comment:
Have you tested any non-default line lengths?

Reply:
Check out these 3 commits on `workflows` - https://github.com/stjudecloud/workflows/compare/main...tmp/format-examples